digitalearthau/runners/celery_environment.py

```diff
@@ -314,18 +320,28 @@ def shutdown():
     return executor, shutdown
 
 
-def _spawn_pbs_workers(redis_host: str, redis_port: str) -> Iterable[subprocess.Popen]:
+def _spawn_pbs_workers(redis_host: str,
+                       redis_port: str,
+                       workers_per_node: int) -> Iterable[subprocess.Popen]:
```

Comment:
Add default value for `workers_per_node=None` with meaning "use all available cores"